packages/react-charting/src/utilities/utilities.ts

```diff
@@ -1082,6 +1162,26 @@ export function getDomainNRangeValues(
       case ChartTypes.LineChart:
         domainNRangeValue = domainRangeOfDateForAreaChart(points, margins, width, isRTL, tickValues! as Date[]);
         break;
+      case ChartTypes.VerticalBarChart:
+        domainNRangeValue = domainRangeOfDateForVerticalBarChart(
```

Comment:
Can't we use the domainRangeOfDateForAreaChart function for vertical bar and vertical stacked use case also.

Reply:
Done